starters/next-wordpress-starter/lib/Menus.js

```diff
@@ -18,11 +18,19 @@ export async function getFooterMenu() {
 		}
 	`;
 
-	const {
-		menu: {
-			menuItems: { edges },
-		},
-	} = await client.request(query);
+	try {
+		const {
+			menu: {
+				menuItems: { edges },
+			},
+		} = await client.request(query);
 
-	return edges.map(({ node }) => node);
+		return edges.map(({ node }) => node);
+	} catch (error) {
+		console.error(
+			'No footer menu data available. Try customizing your query.',
+			error,
+		);
+		return [];
+	}
```

Comment:
I think the error handling could be a bit different as to not spam the console with the stack trace:
Should we link to the new doc you made for this or should we wait until we have a stable URL for our docs site?

```suggestion
	try {
		const { menu } = await client.request(query);
		if (!menu) {
			throw new Error('No footer menu data available. Try customizing your query.')
		}

		return menu?.menuItems?.edges?.map(({ node }) => node);		
	} catch (error) {
		console.error(error.message)
		return [];
	}
```

Reply:
Yeah, that is much more pleasant :) And I think we should wait to link to the docs site until it has a final domain.